docs/scos/dev/front-end-development/202307.0/oryx/building-applications/styling/oryx-responsive-design.md

```diff
@@ -0,0 +1,216 @@
+
+The actual screen definition is provided by a breakpoint configuration and can be referenced by their code. If you need to use the screen definitions in your custom code, you can use `ScreenService` provided by the [layout package](https://www.npmjs.com/package/@spryker-oryx/layout).
+
+{% info_block infoBox "" %}
+
+Screen sizes might be confusing in some cases. For example, when you browse an Oryx application on a wide screen in a smaller browser window, the physical screen size is not relevant because the browser viewport is used to indicate the screen size.
+
+{% endinfo_block %}
+
+## Breakpoints
+
+Breakpoints define the specific screen sizes at which your website or application layout should adapt. Oryx's responsive design system provides default breakpoints for small, medium, and large screen sizes. You can adjust the defaults or introduce custom breakpoints to tailor the design system to your specific needs.
+
+Breakpoints are configurable in themes. Oryx themes come with preconfigured breakpoints, but you can override the breakpoints in an additional theme configuration.
+
+{% info_block infoBox "" %}
+
+It may seem that breakpoints and their usage should be controlled by design tokens and used as CSS variables. Unfortunately, CSS variables can not be used inside media query definitions. This is the main reason why Oryx provides a configurable system to provide the breakpoint definitions.
+
+{% endinfo_block %}
+
+### Default breakpoints
+
+Oryx provides the following default breakpoints for three screen sizes.
+
+| Screen size | Breakpoints                      |
+| ----------- | -------------------------------- |
+| Small       | Smaller than `"768px"`           |
+| Medium      | Between `"768px"` and `"1023px"` |
+| Large       | Larger than `"1023px"`           |
+
+The default breakpoints are given by the `defaultBreakpoints` object, which is available in the [themes](https://www.npmjs.com/package/@spryker-oryx/themes) package. The breakpoint values are configured in pixels.
+
+```ts
+import { Breakpoints, Size } from "@spryker-oryx/utilities";
+
+export const defaultBreakpoints: Breakpoints = {
+  [Size.Sm]: {
+    max: 767,
+  },
+  [Size.Md]: {
+    min: 768,
+    max: 1023,
+  },
+  [Size.Lg]: {
+    min: 1024,
+  },
+};
+```
+
+The breakpoint definition shows that you only need to define the required breakpoint min or max values. For example, there's no `min` value provided for small size, as it defaults to `0`. The same is true for the large screen, which starts with 1024&nbsp;px and supports all screens that are larger than 1024&nbsp;px.
+
+### Custom breakpoints
+
+By using custom breakpoints, you can create a responsive design that is tailored to your project's needs and provides an optimal UX across different devices and screen sizes.
+
+Breakpoints are part of Oryx themes. Oryx themes use the default configuration, but you can configure an additional theme with custom breakpoints.
```

Comment:
Does it mean that to use custom breakpoints, I need to configure an additional theme? Can't really make sense of this paragraph